rbac/server/api/chatbot.py

```diff
@@ -34,14 +34,14 @@
 
 @CHATBOT_BP.websocket("api/chatbot")
 @authorized()
-async def chatbot(request, ws):
+async def chatbot(request, w_s):
```

Comment:
Not sure what w_s is can we have this be what it is?  'write_stream' (If that is what it is supposed to be)

Reply:
Updated to write_stream!